config/yosbr/config/playerex.json

```diff
@@ -0,0 +1,12 @@
+{
+  "resetOnDeath": false,
+  "disableAttributesGui": false,
+  "showLevelNameplates": true,
+  "skillPointsPerLevelUp": 1,
+  "levelFormula": "stairs(x,0.2,2.4,17,10,25)+(0.025x+0.00001x^2)",
```

Comment:
```suggestion
  "levelFormula": "stairs(x,0.2,2.4,17,10,25)",
```

This should stay as default, since it's been a month the server started, and this change *will* affect both the existing and the newcoming players. You don't see an MMORPG going "hey were suddenly changing our hidden formulas so the game is harder for new players now! oh btw there's a small chance youll get an item drop to instantly level up!"

Reply:
If you'd looked at how the graph looks you wouldn't be saying that. The addition does not have a big effect at the lower levels.

Reply:
Where can I see a graph?

Reply:
![image](https://user-images.githubusercontent.com/85982640/202501250-8a2ef5ff-fa52-46cd-b28e-e86d2e727b0c.png)
In desmos maybe, this screenshot does not account for the initial 0-25 roll-in (desmos can't do the stairs function smh) so the Y values are gonna be a bit higher than in reality tho

Reply:
wouldn't this still make it more difficult for everyone else to get to the level you're at?

Reply:
the tomes

Reply:
> the tomes

that really doesn't answer my question

Reply:
The tomes' drop rate is affected by luckiness. Say you have an xp farm that gives you 80 endermen per minute and takes around 5 minutes to get 25 levels at; with luckiness 50 which you'd probably have by the point the increasing level costs start to show you'd have an around 0.1% chance to get a tome from an endermen, that is, around 1000 endermen per tome, which'd be something you'd get every 12-14 minutes. Add this to the levels you get from XP and it compensates out. In addition, you'd still have potential to increase your luckiness and to expand your farm to output more mobs, and that way levelling would be even faster than now.

Reply:
i hope that gets the point across, it's just another meta with more variety and more potential for expansion